ceph-ansible-pipeline/config/definitions/ceph-ansible-pipeline.yml

```diff
@@ -509,13 +509,13 @@
                 condition: SUCCESSFUL
                 execution-type: PARALLEL
                 projects:
-                  - name: 'ceph-ansible-prs-dev-centos7_cluster'
+                  - name: 'ceph-ansible-prs-dev-centos-non_container-cluster'
```

Comment:
Can we we just explicit here and use  'all_daemons' instead of 'cluster' as the scenario?

Reply:
as you pointed out in the ceph-ansible's PR, cluster exists as well